tools/ARIAtools/unwrapStitching.py

```diff
@@ -363,7 +366,7 @@ def __calculateCyclesOverlap__(self):
 
                 # determining the intersection between the two frames
                 if not bbox_frame1.intersects(bbox_frame2):
-                    print("Error, products do not overlap or were not provided in a contigious sorted list.")
+                    log.error("Products do not overlap or were not provided in a contigious sorted list.")
```

Comment:
@Askaholic would you mind suggesting more pythonic way to do this, here as well?